### PR TITLE
Added Format Seconds

### PR DESCRIPTION
**Pull request by RigidStudios**
**Please describe the changes this PR makes and why it should be merged:**

Added a bunch of Time Formats and upped efficiency greatly.

**Status**

- [x] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by Almeeida** [comment]
this update will make the name of this mod misleading

File: actions/convert_seconds_to_days_MOD.js [hunk 1]
```diff
@@ -143,34 +153,68 @@ module.exports = {
...
+              s = time % 60;
+                result = ((h > 0) ? ((h !== 1 ) ? h + " Hours " : h + " Hour ") : "") + ((m > 0) ? ((m !== 1 ) ? m + " Minutes " : m + " Minute ") : "") + ((s !== 1 ) ? s + " Seconds" : s + " Second");
+            break;
+          case 8: // Formatted MS
+              m = Math.floor(time / 60);
+              s = time % 60;
+                result = ((m > 0) ? ((m !== 1 ) ? m + " Minutes " : m + " Minute ") : "") + ((s !== 1 ) ? s + " Seconds" : s + " Second");
+            break;
+          default:
+            print("Something went wrong... Check your Convert Seconds to D/H/M/S mod. (#" + cache.index + ")")
```
Comment (Almeeida) [inline actions/convert_seconds_to_days_MOD.js:215]: undefined variable

Comment (RigidStudios) [reply actions/convert_seconds_to_days_MOD.js:215]: Where exactly?

**Comment by RigidStudios** [comment]
@Almeeida was thinking that as well, should I open a Pull Request, not replacing this mod but as a whole new one?

**Comment by Almeeida** [comment]
> @Almeeida was thinking that as well, should I open a Pull Request, not replacing this mod but as a whole new one? 

Yeha, perhaps that would be the best option

File: actions/convert_seconds_to_days_MOD.js (hunk 1 above)
Comment (Almeeida) [reply actions/convert_seconds_to_days_MOD.js:215]: `print`

**Comment by RigidStudios** [comment]
Changed name, left old mod.

File: actions/convert_seconds_to_days_MOD.js (hunk 1 above)
Comment (RigidStudios) [reply actions/convert_seconds_to_days_MOD.js]: `cache.index` should be defined, and anyways, nobody's actually gonna get this unless they break something.

Comment (Almeeida) [reply actions/convert_seconds_to_days_MOD.js]: "print" is undefined, not talking about cache sir

File: actions/format_seconds.js [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+              s = time % 60;
+                result = ((h > 0) ? ((h !== 1 ) ? h + " Hours " : h + " Hour ") : "") + ((m > 0) ? ((m !== 1 ) ? m + " Minutes " : m + " Minute ") : "") + ((s !== 1 ) ? s + " Seconds" : s + " Second");
+            break;
+          case 8: // Formatted MS
+              m = Math.floor(time / 60);
+              s = time % 60;
+                result = ((m > 0) ? ((m !== 1 ) ? m + " Minutes " : m + " Minute ") : "") + ((s !== 1 ) ? s + " Seconds" : s + " Second");
+            break;
+          default:
+            print("Something went wrong... Check your Format Seconds mod. (#" + cache.index + ")")
```
Comment (Almeeida) [inline actions/format_seconds.js:215]: print isn't defined anywhere.

**Pull request merged**